core/services/ardupilot_manager/main.py

```diff
@@ -15,6 +17,26 @@
 FRONTEND_FOLDER = Path.joinpath(Path(__file__).parent.absolute(), "frontend")
 
 
+class InterceptHandler(logging.Handler):
+    def emit(self, record) -> None:  # type: ignore
+        # Get corresponding Loguru level if it exists
+        try:
```

Comment:
Just to make sure, since we'll use loguru for now on, is this try still necessary ?